api/external/applications/applications.proto

```diff
@@ -212,3 +213,18 @@ enum HealthStatus {
   CRITICAL = 2;
   UNKNOWN = 3;
 }
+
+message Wrapper {
+  string type = 1;
```

Comment:
One case should should think about how we handle, is what happens when Automate is "old" and hab is "new," specifically if a new message type is added and hab starts to send these messages but automate isn't upgraded so it doesn't know what they are. In this demonstration, we would switch the code path based on the content `Wrapper.Type` and drop any messages with unknown values, e.g.,

```go
	switch wrapperMessage.Type {
	case "ExampleOne":
		var fullyUnwrappedMessage applications.ExampleOne
		err = ptypes.UnmarshalAny(wrap1rt.Message, &fullyUnwrappedMessage)
		// errcheck
		handleExampleOneMessage(message)
	case "ExampleTwo":
		// etc.
	default: // drop it
	}
```